### ISSUE TITLE
Provide corresponding aria attributes for the disabled items in the Wizard TOC

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3981]
## Summary

When when the wizard TOC is read by screen readers all the disabled items are 
announced as button disabled.
* What is the change?
1) Remove buttons from wizard TOC and replace them with more appropriate from accessibility perspective markup like menu/menubar/navigation
2) Provide aria-disabled attributes for the disabled items in the TOC
* Why should it go in Clarity?
To provide basic accessibility features
* Does this change impact existing behaviors? If so how?
No
* If this change introduces a new behavior, is this behavior accessible?
Yes

## Use case

As a user of the Clarity wizard I want to be notified by a screen reader that a TOC item is disabled without announcing that the TOC item is button

## Examples

N/A
## Workarounds

N/A

**Comment by gnomeontherun** [comment]
Why would you want to not be told something is a button? It is useful contextually and semantically correct. Some time in the future those buttons may be enabled, otherwise users will only hear text and may not know that the items are in fact interactable. Discoverability is important and semantics make sense here. I don't understand the ask here, so if there is something I'm missing please provide a demo for us to reference.

**Comment by bashibozuk** [comment]
Closing this because it is not present in latest versions of Clarity
